### ISSUE TITLE
tls: failed to find certificate PEM data in certificate input

### ISSUE THREAD

**Issue by johnqa** [alexei-led/pumba#93]
Hi,

I have a Docker Swarm cluster and I generated the client bundle for it.

On my local windows machine I would like to use Pumba to make some chaos testing on the cluster and I am using the command bellow in the folder where all the files are:

**pumba_windows_amd64.exe -H tcp://....:443 --tlsverify --tlscacert ca.pem --tlscert cert.pem**

Unfortunately I am getting this error: **tls: failed to find any PEM data in certificate input**

Can you help me with this? Is there anything I'm missing?

Thank you!

**Comment by alexei-led** [comment]
@johnqa 

1. make usre to pass `--tlskey` (client key for TLS authentication) or have a default key file `/etc/ssl/docker/key.pem`

**Comment by johnqa** [comment]
Hi,

I have tried your suggestions but it didn't work even if I pass the `--tlskey key.pem `

I have tried the command `openssl x509 -in cert.pem -text -noout` and the output looks ok.

Same command line but with docker client (`docker -H tcp://....:443 --tlsverify --tlscacert ca.pem --tlscert cert.pem --tlskey key.pem` ) works but it does take more than 30 seconds to execute.

I have also tried the command on a Linux host.
